src/BBT.Maybe/Maybe.cs

```diff
@@ -15,125 +15,125 @@
     public struct Maybe<T> : ISerializable, IEquatable<Maybe<T>>
         where T : class
     {
-        private readonly T mValue;
+        private readonly T value;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="Maybe{T}"/> struct.
         /// </summary>
-        /// <param name="aValueNullable">The potentially nullable value.</param>
-        internal Maybe(T aValueNullable)
+        /// <param name="value">The potentially nullable value.</param>
+        internal Maybe(T value)
         {
-            this.mValue = aValueNullable;
+            this.value = value;
         }
 
         /// <summary>
         /// Initializes a new instance of the <see cref="Maybe{T}"/> struct.
         /// Constructor used for deserialization.
         /// </summary>
-        private Maybe(SerializationInfo aInfo, StreamingContext aContext)
+        private Maybe(SerializationInfo info, StreamingContext context)
         {
-            MaybeUtils.CheckArgumentNotNull(aInfo, nameof(aInfo));
-            this.mValue = MaybeUtils.GetDeserializedValue<T>(aInfo, nameof(this.mValue));
+            MaybeUtils.CheckArgumentNotNull(info, nameof(info));
+            this.value = MaybeUtils.GetDeserializedValue<T>(info, nameof(this.value));
         }
 
         /// <summary>
         /// Gets a value indicating whether <see cref="Maybe"/> has a value (true)
         /// or is representing the null case (false).
         /// </summary>
-        public bool HasValue => this.mValue != null;
+        public bool HasValue => this.value != null;
 
         /// <summary>
         /// Checks whether the operands are equal.
         /// </summary>
-        /// <param name="aA">Maybe to compare.</param>
-        /// <param name="aB">Maybe to compare.</param>
-        public static bool operator ==(Maybe<T> aA, Maybe<T> aB)
+        /// <param name="a">Maybe to compare.</param>
+        /// <param name="b">Maybe to compare.</param>
+        public static bool operator ==(Maybe<T> a, Maybe<T> b)
         {
-            return object.Equals(aA.mValue, aB.mValue);
+            return object.Equals(a.value, b.value);
         }
 
         /// <summary>
         /// Checks whether the operands are unequal.
         /// </summary>
-        /// <param name="aA">Maybe to compare.</param>
-        /// <param name="aB">Maybe to compare.</param>
-        public static bool operator !=(Maybe<T> aA, Maybe<T> aB)
+        /// <param name="a">Maybe to compare.</param>
+        /// <param name="b">Maybe to compare.</param>
+        public static bool operator !=(Maybe<T> a, Maybe<T> b)
         {
-            return !(aA == aB);
+            return !(a == b);
```

Comment:
`return !(a == b);` may be simplified by `return a != b;`

Reply:
Done.